robocop/checkers/naming.py

```diff
@@ -644,7 +643,6 @@ class DeprecatedStatementChecker(VisitorChecker):
 
     reports = ("deprecated-statement",)
     deprecated_keywords = {
-        4: {"runkeywordunless": "IF"},
```

Comment:
It's fine fix for now. But I see the problem (and that's why the bug was raised). If the RF 6 will be released this report will stop displaying deprecation messages even if the keyword was deprecated in RF 5. Perhaps it could be reverse: the deprecated statements should be keys, and values would be (alternative, version). That way you would just check if statement is deprecated, and if get value to display message.

Reply:
Then we of course would need to perform extra check if actual version is equal or bigger than stored version.

Reply:
Are we going to merge this one or it should include your proposed changes?

Reply:
We can merge this one. Proposed change can go in separate PR in the future - we have time before 6.0 release.